generators/server/templates/src/main/resources/config/_application-dev.yml

```diff
@@ -25,6 +25,8 @@ eureka:
 <%_ } _%>
 
 spring:
+    profiles:
+        active: dev,swagger
```

Comment:
@deepu105 so it looks like you are OK to merge this - I'm doing it. And then I'm going to look at the other issue.